docs/writing-rules/private-rules.md

```diff
@@ -70,7 +71,9 @@ To enforce the rule on new scans, add the rule in the [registry](https://semgrep
 
 ## Automatically publishing rules
 
-This section provides an example of how to automatically publish your private rules so they are accessible within your private organization. "Publishing" your private rules in this manner does not make them public. The following sample of the GitHub Actions workflow publishes rules from a private Git repository after a merge to the `main`, `master`, or `develop` branches.
+This section provides examples of how to automatically publish your private rules so they are accessible within your private organization. "Publishing" your private rules in this manner does not make them public. In these examples, the private rules are stored in `private_rule_dir`, which is a subdirectory of the repository root. If your rules are in the root of your repository, you can replace the command with `semgrep publish --visibility=org_private .` to refer to the repository root. You cannot omit the directory specification.
```

Comment:
I'm not sure if putting it in that specific way is more helpful in the context of a CI action. That instruction makes sense on the CLI, but it doesn't give you much in CI, since CI is going to do things in the repository root unless instructed otherwise.

We could create examples for both the root and subdirectory cases, but I wonder if it's overkill. Or do you have a different idea of what we could do?

Reply:
No, let's leave it -- I was definitely thinking of this in terms of the CLI, but given the limitations of CI, let's leave things as it